src/patternfly/components/Card/card-title.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-card__title{{#if card-title--modifier}} {{card-title--modifier}}{{/if}}"
+  {{#if card-title--attribute}}
+    {{{card-title--attribute}}}
+  {{/if}}>
+  {{> @partial-block}}
+</div>
```

Comment:
I wonder about changing this to `__title` for a couple of reasons. It's somewhat confusing, without supporting examples, as to when I should use `h2.pf-c-card__title` vs `.pf-c-card__title > h2.pf-c-title.pf-m-xl` (if I want to change the card header/title text styling). And, because of React's implementation, they'll need to make API changes. Should you revert this and open a follow up issue?

Reply:
re: examples, agree but that's pre-existing. All of our docs/examples use a div, and the react `<CardHeader>` just renders a `<div>`. And in react, the `<CardHeader>` isn't a `<Title>` component, it just hardcodes the `.pf-c-title.pf-m-md` classes to achieve a font-size/font-weight combo. The goal of this change is just to remove the dependency on `.pf-c-title` being hardcoded as a class to `.pf-c-card__header`, and apply the font size/weight combo to the card element instead. 

If we just remove `.pf-c-title` from `.pf-c-card__header`, and style that element directly, I'm pretty sure that doesn't break the react API. React just needs to remove the hard coded class from the [CardHeader component](https://github.com/patternfly/patternfly-react/blob/v4/packages/react-core/src/components/Card/CardHeader.tsx). But agreed I'll revert this - I'd like to verify to be sure, and it's past the point where we can add work to react's plate. Also, I'd like to rename it to `card__title` to be consistent (and not confuse with `card__head`).

I made a similar change to the empty state component in this PR I'll revert, too. We have an issue filed in react to update instances like the card title and empty state title that I think we could get this work in with since it has been planned - https://github.com/patternfly/patternfly-react/issues/3676